package.json

```diff
@@ -21,14 +21,14 @@
     "license-checker": "^25.0.1",
     "lodash": "^4.17.21",
     "semver": "^7.3.5",
-    "spdx-license-ids": "^3.0.7",
-    "spdx-satisfies": "^5.0.0",
-    "yargs": "^16.2.0"
+    "spdx-license-ids": "^3.0.9",
+    "spdx-satisfies": "^5.0.1",
+    "yargs": "^17.0.1"
```

Comment:
It does not look like we'll be affected by the breaking changes: https://github.com/yargs/yargs/releases/tag/v17.0.0